exporter/prometheusremotewriteexporter/metadata.yaml

```diff
@@ -27,3 +27,17 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    exporter_prometheusremotewrite_consumers:
+      enabled: true
+      description: Number of configured workers to use to fan out the outgoing requests
+      unit: "1"
+      gauge:
```

Comment:
This needs to be an updown counter

Reply:
real question, What's the difference between gauge and updown counter?

Reply:
I had the same question and ended up [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#gauge).

I think the answer is: if the `sum` of values across a dimension makes sense then it should be an up and down counter.  In this case, it might make sense to sum the number of consumers across several collectors sending to the same endpoint to try and identify what kind of concurrency you have against your metric storage so I think it makes sense.

Reply:
@dashpole I have adressed your feedback in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37912/commits/62bcf74b1d50eb487fe26481f73b173b42e1d15d

Reply:
It mostly has to do with how they are aggregated by defalt.  UpDownCounters aggregate by summing.  Gauges aggregate by averaging (or computing other things about the distribution).

An example of an UpDownCounter is memory usage.  You could reasonably measure total memory usage across a number of pods by summing. But you would never want to sum a temperature, so that is a gauge.

Reply:
Thank you both for the explanation ❤️